tests/queries/0_stateless/02864_statistic_exception.sql

```diff
@@ -2,29 +2,29 @@ DROP TABLE IF EXISTS t1;
 
 CREATE TABLE t1 
 (
-    a Float64 STATISTIC(tdigest),
-    b Int64 STATISTIC(tdigest),
+    a Float64 STATISTICS(tdigest),
```

Comment:
Maybe change the file names of the .sql and .reference files from `statistic` to `statistics`?